File: python/basics/loops.py
# while loop
maximum = 100
position = 1
while position < maximum:
    position += 1

    if position == 11:
        pass

    if position == 22:
        # skip the whole remaining body
        continue

    print('Current index: {}'.format(position))

    if position > 55:
        # Stop loop at position > 55
        break


# For loop
numbers = [1, 2, 3, 6, 7, 9]
for n in numbers:
    print('Current number: ', n)


age = 30
for _ in range(10):
    print('Hello World!')
    if age == 30:
        # break the whole for loop
        break
    if age > 10:
        # skip the remaining body
        continue
    print('')

File: javascript/basics/loops.js
// loops and conditions:
// for loop
const numbers = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9]
for(var i=0; i<numbers.length; i++){
    if(numbers[i] % 2 === 0){
        console.log(numbers[i], 'is even')
    }else{
        console.log(numbers[i], 'is odd')
    }
}

// do .. while loop:
const limit = 100
do{
    console.log('First Limit: ', limit)
}while (limit > 0) {
    console.log('Current Limit: ', limit)
    limit -= 20
}